views/vwLearner/homeRevision.hbs

```diff
@@ -57,7 +93,7 @@
   </div>
 
   <div class="review mb-5">
-    <p class="review-test-text">Today review test has {{this.totalWords}} words</p>
+    <p class="review-test-text">Today review test has {{this.reviewWordsCount}} words</p>
```

Comment:
chỗ này cũng cho vào `messages.js` được lun, giữ nguyên cái {{}} thì lúc render ra vẫn gọi được biến `this.reviewWordsCount`, tui nhớ là vậy

Reply:
ko lun ô